code-challenges.test.js

```diff
@@ -57,6 +125,38 @@ var hand2 = [5, 5, 3, 3, 4]
 var hand3 = [5, 5, 5, 5, 4]
 // Expected output: false
 
+describe('fullHouse', ()=>{
+    it('returns a boolean value if an array is a full house or not', ()=>{
+      expect(fullHouse(hand1)).toEqual(true)
+      expect(fullHouse(hand2)).toEqual(false)
+      expect(fullHouse(hand3)).toEqual(false)
+    })
+  })
 
+  //cool got red
 
 // b) Create the function that makes the test pass.
+
+//create a function that takes in an array
+//iterate through array to count instances of a number/value
+//use conditional if 3 instances of a number && 2 instances of a number return true
+//else return false
+//got stuck with this one. I wanted to use an increment to count the instances and then use that to
+//create a conditional that would return a boolean value, but after several attempts I couldn't get it. However, I was experimenting with this solution. I did ask for help from fellow classmates that were available (not in 2021d) chat, but those few weren't successful either
+
+    
+    const fullHouse = arr => {
+        const copy = arr.slice();
+        for(let i = 0; i < arr.length; ){
+           const el = copy.splice(i, 1)[0]
+           if(copy.includes()){
+              copy.splice(copy.indexOf(), 1)
+              if(copy.includes()){
+                 return true;
+              }
+           }else{
+              i++
+           }
+        }
+        return false
+     }
```

Comment:
You've got a lot of moving parts here. I think step one is separating the lines out and exploring what each thing is trying to do. If we log (using jest) the variable copy on the line after it's called with slice() what do you get? make an adjustment to the code log it; rinse and repeat.

Reply:
Let me know if you want to work through this together